tenzir/functional-test/tests/vast.bats

```diff
@@ -198,20 +183,10 @@ teardown() {
 
 #bats test_tags=import,json,suricata
 @test "Import time" {
-  if [ $(uname) == "Darwin" ]; then
-    # TODO: Figure out correct invocation of `date` for mac
-    skip "disabled on mac"
-  fi
-
   import_suricata_eve 'where #schema == "suricata.stats"'
 
-  # We need subsecond precision here because `date` rounds
-  # down otherwise. Also, we need `tr` because `date` uses
-  # a comma for subsecond precision by default but tenzir
-  # requires a dot. (ISO 8601 allows both)
-  NOW=$(date -Ins | tr ',' '.')
-  check tenzir "export | where #import_time > ${NOW}"
-  check tenzir "export | where #import_time <= ${NOW} | sort timestamp"
+  check tenzir "export | where #import_time > now"
+  check tenzir "export | where #import_time <= now | sort timestamp"
```

Comment:
I think these are nice additional test cases, but it's not quite the same as the original test which verifies that the assigned import time is consistent with the system time.

Reply:
I pushed a fix for this so that it runs correctly on macOS.